content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -1235,7 +1235,7 @@ contributors. Follow the instructions in the SIG-Windows
 * `kubectl port-forward` fails with "unable to do port forwarding: wincat not found"
 
   This was implemented in Kubernetes 1.15 by including wincat.exe in the
-  pause infrastructure container `mcr.microsoft.com/oss/kubernetes/pause:3.4.1`.
+  pause infrastructure container `k8s.gcr.io/pause:3.5`.
   Be sure to use these versions or newer ones.  If you would like to build your
   own pause infrastructure container be sure to include
   [wincat](https://github.com/kubernetes-sigs/sig-windows-tools/tree/master/cmd/wincat).
```

Comment:
This isn't the source for wincat in `k8s.gcr.io/pause:3.5`. It comes from https://github.com/kubernetes/kubernetes/tree/master/build/pause/windows/wincat now.